README.md

```diff
@@ -73,9 +71,15 @@ as `error` level logs.
 
 ##### Methods
 
-#### `logger.report(level, report[, state])`
+#### `logger.info(action[, state])`
+
+#### `logger.warn(action[, state])`
+
+#### `logger.error(action[, state])`
+
+#### `logger.debug(action[, state])`
 
-`logger.report(level, report[, state])` is a function which is used to register a new log in the buffer.
+#### `logger.emerg(action[, state])`
```

Comment:
i think `critical` is a more commonly used term

Reply:
i was just being consistent with bunyun-syslog's terminology.

Reply:
ok thats cool too